courses/serializers/v1/base.py

```diff
@@ -36,6 +36,15 @@ class Meta:
 class BaseCourseRunSerializer(serializers.ModelSerializer):
     """Minimal CourseRun model serializer"""
 
+    is_archived = serializers.SerializerMethodField()
+
+    def get_is_archived(self, instance):
+        return (
+            instance.is_enrollable
+            and instance.enrollment_end is None
```

Comment:
In the [`get_archived_courseruns` function,](https://github.com/mitodl/mitxonline/pull/2359/files#diff-1891631991481dca5fef7d88592f36f41fe1422ddc3f13e172d8973bc77c8da6R181) we defined enrollment end as able to be null or future.  Here we're ONLY defining it as None. Is this the case? Should the other function be updated? Should this one?

Reply:
that condition is not needed there at all.